### ISSUE TITLE
Check all pages are accessible (needed for Beta)

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeBMD2#466]
The Documentation Team is working on Accessibility guidance for Developers, to help address the following items which may need attention in FreeBMD2. Once complete, they can be checked off the list.

The below items are stories made from the Accessibility audit on FreeREG; the CLOSED ones are not necessarily complete in FreeBMD2.

- [ ] **STAS-F10: Non-Descriptive Page Titles** #249Systemwide OPEN
- [ ] **STAS-F03: Empty Links** : need to check that the empty links in headings have been fixed.
- [ ] **STAS Review and adjust the search results page for accessibilty** Accessibility Review Actor: researcher OPEN
- [ ] **Review and adjust the search form page for accessibility** Accessibility Review Actor: researcher IN PROGRESS
- [ ] **STAS Review and adjust the search results detail page for accessibility** Accessibility Review Actor: researcher  OPEN
- [ ] **STAS-F07: Areas Inaccessible via Keyboard** Accessibility Review HTML @PatReynolds to try and recruit a volunteer family historian who can pick up any issues that are missed and report in via 'report a problem', email, or other way they find easiest.

*****************************************

- [x] **STAS Review and adjust the about search results page for accessibility** Accessibility Review ready - now part of Help #965 1.7 Researchers Product Backlog

- [ ] **STAS-F01: Missing ‘Skip’ Navigation** Accessibility Review P0 ready #10 1.4 Researchers Product Backlog NEEDS #153Systemwide

- [x] **STAS-F01: Opening in New tab issues  #467 #488 #369

--------------Reviewed to this Point 28 July 2022-------------------------

- [ ] **STAS-F02: Incorrect Heading Structure** Accessibility Review HTML P0 ready public pages footer 

- [ ] **STAS-F04: Inappropriate use of Fieldsets** Accessibility Review CSS FORM P0 ready #920 1.4 Researchers Product Backlog

- [ ] **STAS-F05: Missing Form Labels** Accessibility Review CSS FORM P0 ready #921 1.4 Researchers Product Backlog

- [ ] **STAS-F06: Orphaned Form Label** Accessibility Review CSS FORM P0 ready #922 1.4 Researchers Product Backlog

- [ ] **STAS-F08: HTML Markup Errors** Accessibility Review HTML P0 ready #924 1.4 Researchers Product Backlog

- [ ] **STAS-F09: Duplicate ID’s** Accessibility Review CSS FORM P0 ready #925 1.4 Researchers Product Backlog

- [ ] **STAS-F11: Non-Descriptive Alt Text** Accessibility Review HTML P0 ready #927 1.4 Researchers Product Backlog

- [ ] **STAS-F12: Data Tables with Incorrect Markup** Accessibility Review CSS HTML P0 ready #928 1.4 Researchers Product Backlog

- [ ] **STAS-F13: Illogical Tab Order** Accessibility Review HTML P0 ready #929  1.4 Researchers Product Backlog

- [ ] **STAS-F14: Links not Visible in Focus** AA Accessibility Review CSS P0 ready #930  1.4 Researchers Product Backlog

- [ ] **STAS-A02: Visual Cue on Keyboard Focus** Accessibility Review CSS P0 ready #932 1.4 Researchers Product Backlog

- [ ] **STAS-A03: Links not Understood out of Context** Accessibility Review HTML P0 ready #934 1.4 Researchers Product Backlog

- [ ] **STAS-A04: Users not notified of Links Opening in a new Window** Accessibility Review HTML P0 ready #935 1.4 Researchers Product Backlog

- [ ] **20 STAS-F15: Insufficient Colour Contrast** AA Accessibility Review CSS P20 Systemwide epic in progress #931 1.4 Researchers On Ice

- [ ] **STAS-A01: Accessibility Statement (Review and Update)** Accessibility Review Actor: executive P11 documentation ready #933 1.4 Researchers Product Backlog

**Comment by KristinaGadzhieva** [comment]
@nurifreembd  could you please review the list above and compare it with the results of analysis in #617?
